src/lib/index.js

```diff
@@ -149,11 +149,18 @@ define("WherebyEmbed", {
                 this.roomUrl.searchParams.set(k, v);
             }
         });
+
+        const isLocalRecordingEnabled = this.roomUrl.searchParams.has("recording");
+
+        if (isLocalRecordingEnabled)
+            this.iframe.sandbox = "allow-downloads allow-same-origin allow-forms allow-scripts allow-popups";
```

Comment:
ps. I think this should be `this.iframe.current.sandbox`, perhaps with some conditional checks in there first
<img width="562" alt="Screenshot 2023-11-21 at 17 04 31" src="https://github.com/whereby/browser-sdk/assets/117101145/80dda019-9eb4-4c08-b1f4-7226b39c3dec">
<img width="582" alt="Screenshot 2023-11-21 at 17 04 25" src="https://github.com/whereby/browser-sdk/assets/117101145/20f1e9f2-6c12-4cc8-9c23-08569a12340e">